baldrick/plugins/milestone_checker.py

```diff
@@ -0,0 +1,33 @@
+from flask import current_app
```

Comment:
Since this is a github plugin, maybe it should be called ``github_milestone_checker``